kubernetes/aws-eks/blueprint.yaml

```diff
@@ -17,7 +18,7 @@ inputs:
 
   aws_region_name:
     type: string
-    default: 'us-east-1'
+    default: 'eu-west-1'
```

Comment:
We actually should use us-east-1, because this is the most common location.

Reply:
I know but something happened yesterday(capacity issue) and it didnt work on us-east-1 region . wierd.. but i can change it back